pagopa/io-app
# fix: [IOBP-908] Timeline amount format in cents

## Short description
This PR fixes the timeline amount format considering the amount as cents

## List of changes proposed in this pull request
- Used the `formatNumberCentsToAmount` function inside the `formatAbsNumberAmountCentsOrDefault` to format the amount from cents into correct string.

## How to test
- The related unit tests should pass correctly
- Checkout the dev-server with this PR: https://github.com/pagopa/io-dev-api-server/pull/416
- After that enable IDPay feature flag from the profile settings
- With the dev-server started you should be able to see into "Portafoglio" section two mocked initiatives
